libcontainer/intelrdt/intelrdt.go

```diff
@@ -294,8 +294,11 @@ func getIntelRdtRoot() (string, error) {
 }
 
 func isIntelRdtMounted() bool {
-	_, err := getIntelRdtRoot()
-	return err == nil
+	if _, err := getIntelRdtRoot(); err != nil {
```

Comment:
This hunk looks weird. This was just fixed in commit e8eb8000f15 and it seems you're reverting the change by mistake.